### ISSUE TITLE
The psychometric properties of the Mathematics Attitude Scale for Adults (MASA)

### ISSUE THREAD

**Issue by ben-domingue** [ben-domingue/irw#240]
https://link.springer.com/article/10.1007/s12144-022-02980-9

**Comment by ben-domingue** [comment]
see osf site.

**Comment by ben-domingue** [comment]
osf site. https://osf.io/m593n/

this one is fairly easy i just think we'll need to talk about how many scales to report. we could do:
- 7 (one for each)
- 2 (math v polish)
- 1

let's talk about this

**Comment by ben-domingue** [comment]
great work! 
1. i think we should probably include this single item as a covariate with the other data? or we could lump it in with amas? maybe the latter is preferable but i'm also open to @SamuelEnrique thoughts. we've avoided single-item scales so i don't think we want it alone but i am flexible as to how we include. 
2. excellent decision. great.
3. sounds good

really nice work here!

**Comment by SamuelEnrique** [comment]
@ben-domingue good point; I commented earlier on the spreadsheet because I couldn't find it! Updated the datasets, did a new PR, and finished tags/dictionary (except for that one comment about the license). 

Just to close the loop on number one earlier, I added SIMA as a covariate since it fits that category more than MASA/AMAS (by far, in my opinion, but we can chat on this as well).
